packages/rancher-istio/charts/README.md

```diff
@@ -1,40 +1,54 @@
-# Rancher Istio Installers
+# Rancher-Istio Chart
 
 A Rancher created chart that packages the istioctl binary to install via a helm chart.
```

Comment:
lets copy the other one here
```suggestion
Rancher's Istio installer wraps the istioctl binary commands in a handy helm chart, including an overlay file option to allow complex customization.
```

Reply:
I have copied this over in the latest commits.